atomic_client.py

```diff
@@ -109,8 +126,20 @@ def print_scan_list(all_scanners):
             print(resp)
 
         elif(sys.argv[1] == "scan"):
-            if(sys.argv[2] == "list"):
+            if("--list" in sys.argv):
                 all_scanners = json.loads(dbus_proxy.scan_list())
                 print_scan_list(all_scanners)
+
```

Comment:
This should be just a switch statement.  We don't want to allow users to specify multiple commands in sys.argv.